lua/autorun/client/cl_contact_forms.lua

```diff
@@ -810,4 +810,32 @@ net.Receive( "CFC_ContactForms_FailureAlert", function()
     )
 end )
 
+net.Receive("cfc_contact_forms_alert", function()
+    local data = net.ReadTable()
+    local reporter = player.GetBySteamID(data["steam_id"])
+    local reported = player.GetBySteamID(data["reported_steam_id"])
+
+    local reporterName = reporter:GetName() -- reporter's name
+    local reporterRankColor = team.GetColor(reporter:Team()) -- color of reporter's rank
```

Comment:
```suggestion
    local reporterName = reporter:GetName()
    local reporterRankColor = team.GetColor(reporter:Team())
```
comments like this dont help anyone read or understand the code so they can be removed.

for example:
if the variable is already called reporterName we dont need a comment `-- reporter name`

Reply:
Removed comments.